Dockerfile_init

```diff
@@ -20,6 +20,7 @@ RUN wget -O ephemeris.tar https://api.github.com/repos/galaxyproject/ephemeris/t
 
 #COPY config/galaxy.ini config/galaxy.ini
 COPY config/job_conf.xml config/job_conf.xml
+RUN sed -i 's/HANDLERS -->//' config/job_conf.xml && sed -i 's/<!-- HANDLERS//g' config/job_conf.xml
```

Comment:
So these are now correct here ? 
```
pablo Jun 12th at 4:53 PM
Ok, I think I have found the problem... I used to have some sed lines on the Dockerfile to switch handlers for the old setup and the new setup....…
```

Reply:
Yes, this lines are correct now... however, image generation will still fail due to https://github.com/bgruening/docker-galaxy-stable/issues/441

We could attempt a fix on our own fork of docker-galaxy-stable.